app/models/reaction.rb

```diff
@@ -264,7 +264,12 @@ def cleanup_array_fields
   def auto_set_short_label
     prefix = creator.reaction_name_prefix
     counter = creator.counters['reactions'].succ
-    self.short_label = "#{creator.initials}-#{prefix}#{counter}"
+    sh_label = self['short_label']
+    return if short_label && !short_label_changed?
```

Comment:
not needed imo

Reply:
agreed, I'll remove once the logic discussion is resolved.